bears-app/src/components/shared/api/mock-data/content-data.json

```diff
@@ -0,0 +1,814 @@
+{
+  "lifeEventForm": {
+    "id": "death-of-a-loved-one",
+    "timeEstimate": "60 minutes",
+    "titlePrefix": "",
+    "title": "Benefit Finder After Losing A Loved One",
+    "summary": "We are sorry about your loss. You and your family may be eligible for survivor benefits from the government following a family member's death. Learn how to find the benefits you may qualify for.",
+    "lede": "",
+    "secondaryHeading": "Steps to find benefits",
+    "icon": "",
+    "sectionsEligibilityCriteria": [
```

Comment:
this is made up of an array of criteria sections, each section is a step that follows the inital component of the life event title, description, etc.
<img width="631" alt="Screenshot 2023-06-27 at 12 17 34 PM" src="https://github.com/GSA/px-bears-drupal/assets/37077057/2d61e044-b46a-432a-ae79-6dd63f82db17">